src/Internal/Wallet/Cip30/SignData.js

```diff
@@ -0,0 +1,111 @@
+/* global BROWSER_RUNTIME */
+
+let lib;
+if (typeof BROWSER_RUNTIME != "undefined" && BROWSER_RUNTIME) {
+  lib = require("@emurgo/cardano-message-signing-asmjs");
```

Comment:
I think we want https://www.npmjs.com/package/@emurgo/cardano-message-signing-browser and not the asmjs version (very slow).